### DISCUSSION TITLE
FBW A32NX EFB charts stop working after proceeding with my flight

### DISCUSSION THREAD

**Discussion by nantucket01** [flybywiresim/aircraft#6273] [Help]
I am using the Stable version ( 7.3 ) for the FBW A32NX aircraft. I have a subscription to Navigraph and begin looking up various charts at my departure airport. Everything works perfectly until I get approximately mid-flight and begin searching for charts for my destination airport. The EFB freezes and will not display any charts or recognize the ICAO that I type in the search bar. Any help in resolving this issue will be appreciated!!

**Comment by f1r3punch** [comment]
I have same issue on Dev version... once you take off...  no access to Navigraph via EFB, doesn't search the airports. 
Restarting EFB doesn't help. 

Navigraph in-game panel works fine, the entire time.

**Reply by nantucket01** [reply]
Not sure if this was overlooked, but we have not received an answer on this problem?
